Viper Practice/src/helpers/Localization/Strings.swift

```diff
@@ -0,0 +1,180 @@
+//
+//  Strings.swift
+//  Viper Practice
+//
+//  Created by Huzaifa-Umer on 18/04/2024.
+//
+
+import Foundation
+import CoreText
+
+struct Strings {
+    
+    struct Button {
+
+        static var settings: String {
```

Comment:
we can reduce the extra number of lines by following:
 static let settings = "Settings"
 
 Also, we don't need var here

Reply:
removed extra string